django_project/frontend/api_views/data_table.py

```diff
@@ -0,0 +1,28 @@
+from frontend.filters.data_table import OwnedSpeciesFilter
+from frontend.serializers.data_table import OwnedSpeciesSerializer
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from species.models import OwnedSpecies
+
+
+class DataTableAPIView(APIView):
+    permission_classes = [IsAuthenticated]
+    filter_class = OwnedSpeciesFilter
+
+    def get_queryset(self):
+        organisation_id = self.request.session.get('current_organisation_id')
+        queryset = OwnedSpecies.objects.select_related('user').filter(
+            user__organisationuser__organisation_id=organisation_id
```

Comment:
@ketankartoza query would be the OwnedSpecies that are in property belongs to current organisation. That way you can get all properties inside organisation.

Reply:
Like this?
queryset = OwnedSpecies.objects.filter(property__organisation_id=organisation_id

Reply:
yes, that would be correct @ketankartoza

Reply:
okay

Reply:
@ketankartoza the queryset is still not changed to use organisation from property